### DISCUSSION TITLE
IMPORT PCAP FILE

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#7696] [Unsupported Versions] [answered]
I have successfully imported the pcap file, however, nothing comes up in Hunt, I have also selected the relevant date range.
![import](https://user-images.githubusercontent.com/6872571/161485689-9d8e63fc-1561-4b2c-90ed-a439d049e665.JPG)

What could be wrong?

**Comment by tomjones1977** [comment, marked as answer]
Check the Zeek logs in Kibana - can you see any data in there for the pcap time range?

**Reply by ekky19** [reply]
Thank you, man, 
I saw the Logs in Kibana, and right after that saw them in Hunt and Alerts too.

**Comment by ekky19** [comment]
I have just imported the following pcap file... now again nothing comes up in Alerts. Apart from "Alerts" logs come up in other places.

https://blog.securityonion.net/2022/06/quick-malware-analysis-emotet-epoch-5.html?m=1

What could be wrong? @dougburks 

Also in Version: 2.3.130 I had to close the firewall and open it again to access to web interface... didn't happen in previous versions...
sudo systemctl stop firewalld
sudo systemctl start firewalld

**Reply by dougburks** [reply]
You shouldn't need to restart firewalld manually. This might indicate that your installation didn't complete successfully (which might also have an effect on importing pcaps). You might take a look at the sosetup.log file for any clues and consider performing a new installation. You can also try the steps here:
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts

**Reply by dougburks** [reply]
> What could be wrong? @dougburks

Please note the following from https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
_Tagging Individuals
Please do not tag an individual in a discussion unless that individual has already volunteered to help you in that discussion._

**Reply by ekky19** [reply]
Thank you.
